.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @idea-pool/idea-pool-staff-team
```

Comment:
Open to suggestions for the CODEOWNERS content. Should work with users & Github Groups.
Although for some reason the `@idea-pool/idea-pool-js-assertion-cheat-sheet` group is not recognised by GitHub 😞.

Reply:
Please use `@idea-pool/js-assertion-cheat-sheet-maintainers`: https://github.com/orgs/Idea-Pool/teams/js-assertion-cheat-sheet-maintainers